### ISSUE TITLE
Does not build manpages with pandoc 2.2.1

### ISSUE THREAD

**Issue by yaccz** [neithernut/git-dit#185]
```
error: failed to run custom build command for `git-dit v0.4.0 (/home/yac/git/3rd/git-dit)`
process didn't exit successfully: `/home/yac/git/3rd/git-dit/target/debug/build/git-dit-627154aff84fc88d/build-script-mkmanpage` (exit code: 101)
--- stderr
--smart/-S has been removed.  Use +smart or -smart extension instead.
For example: pandoc -f markdown+smart -t markdown-smart.
Try pandoc --help for more information.
thread 'main' panicked at 'assertion failed: Command::new("pandoc").arg("-s").arg("-S").arg("-f").arg("markdown").arg("-t").arg("man").arg("git-dit.1.md").arg("-o").arg("git-dit.1").status().unwrap().success()', mkmanpage.rs:5:9
note: Run with `RUST_BACKTRACE=1` for a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by neithernut** [comment]
Thanks for the report. Will fix as soon as I'm back from work.

The fix itself will be trivial, but I'll have to figure out whether old pandoc versions support the "new" markdown format and whether or not we'll simply drop support for those versions. For example, Ubuntu-18.04 (a LTS release) still ships pre-2.0 pandoc (pandoc-2.0 [was released in Sep. 2017](https://pandoc.org/releases.html#pandoc-2.0-29-oct-2017)).

**Comment by neithernut** [comment]
I pushed a tentative fix (commit 2b9e660ec80d52d3e009305cd671bde7c846f59d) which simply removes the `-S` option. Unfortunately, I can't test the fix with a newer `pandoc`. Turns our my distro also ships a completely outdated version.

**Comment by yaccz** [comment]
Seems to work correctly. I got a lone git-dit.1 in the project root. It can be viewed with man and looks reasonable.
